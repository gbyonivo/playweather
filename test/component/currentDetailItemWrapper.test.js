import React from 'react';
import renderer from 'react-test-renderer';
import CurrentDetailItemWrapper from '../../src/components/currentDetailItemWrapper';

describe('CurrentDetailItemWrapper component', () => {
  it('renders correctly', () => {
    const actual = renderer.create(<CurrentDetailItemWrapper headerText={'playweather'} children={<div/>} />);
    expect(actual).toMatchSnapshot();
  });
});