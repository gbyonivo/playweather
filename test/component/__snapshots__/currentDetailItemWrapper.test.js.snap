// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentDetailItemWrapper component renders correctly 1`] = `
<div
  className="currentDetailItemWrapper"
>
  <h2>
    playweather
  </h2>
  <div />
</div>
`;
